### PR TITLE
Feature/oracle schema support

### PR DESCRIPTION
**Pull request by vkirichenko**

### CODE REVIEW

File: src/main/resources/config/liquibase/changelog/20170714230000_modify_login_size_and_uniq.xml [hunk 1]
```diff
@@ -31,17 +31,18 @@
...
 
         <addUniqueConstraint columnNames="email"
                              tableName="jhi_user"/>
 
-        <createIndex indexName="idx_user_login"
-                     tableName="jhi_user"
-                     unique="true">
-            <column name="login" type="varchar(100)"/>
-        </createIndex>
+        <!--<createIndex indexName="idx_user_login"-->
```
Comment (ishkurko) [inline src/main/resources/config/liquibase/changelog/20170714230000_modify_login_size_and_uniq.xml]: Could we remove this tags?

File: src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml [hunk 2]
```diff
@@ -34,11 +38,11 @@
                                  referencedTableName="jhi_user"/>
 
 
-        <createIndex indexName="idx_user_key"
-                     tableName="jhi_user"
-                     unique="true">
-            <column name="user_key" type="varchar(255)"/>
-        </createIndex>
+        <!--<createIndex indexName="idx_user_key"-->
```
Comment (ishkurko) [inline src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml]: Could we remove this tags?

File: src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml [hunk 3]
```diff
@@ -69,13 +73,14 @@
...
-
-        <createIndex indexName="idx_user_login"
-                     tableName="user_login"
-                     unique="true">
-            <column name="login" type="varchar(255)"/>
-        </createIndex>
+        <addUniqueConstraint tableName="user_login" columnNames="login"
+                             constraintName="idx_user_login"/>
+
+        <!--<createIndex indexName="idx_user_login"-->
```
Comment (ishkurko) [inline src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml]: Could we remove this tags?

File: src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml (hunk 2 above)
Comment (vkirichenko) [reply src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml]: removed

File: src/main/resources/config/liquibase/changelog/20170714230000_modify_login_size_and_uniq.xml (hunk 1 above)
Comment (vkirichenko) [reply src/main/resources/config/liquibase/changelog/20170714230000_modify_login_size_and_uniq.xml]: removed

File: src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml (hunk 3 above)
Comment (vkirichenko) [reply src/main/resources/config/liquibase/changelog/20170818133100_multilogin_model.xml]: removed

**Pull request merged**